experimental/generation/generator/README.md

```diff
@@ -9,220 +9,62 @@ Generate Bot Framework Adaptive Dialogs from JSON schema.
-    - Look for `<propertyType>.lg` which usually defines `# entities` and `#
-      templates`.
-    - Per-template in the property schema `$templates` or in `# templates`
-      generate .lg, .lu, .qna, .dialog files.
-  - Evaluate templates defined in the schema root level `\$templates` of the
-    merged schema.
-- Evaluate schema expressions and write the full schema out to
-  `<prefix>.schema.dialog`.
-
-All generated assets are named using a standard pattern which is what enables
-updating generated files when changing the schema. In the below, $italics$ are
-place holders. 
-* $schema$-library -- Defines library files that are either common building
-  blocks like [library.lg.lg](templates/library.lg.lg) or are internal
-  mechanisms like [library-help.lg.lg](templates/library-Help.lg.lg).
-* $schema$-$entity$Entity -- Defines entity specific files.
-* $schema$-$property$ -- Defines property related files when $property$ is found
-  in the schema.
-* $locale$/ -- Assets for a particular locale that follows the above naming
-  patterns and adds $locale$ to the file extension, an example would be
-  `en-us/sandwich-Bread.en-us.lg` which would be a localized .lg asset for the
-  `Bread` property in the `sandwich` schema. You can define your own templates
-  that add to the naming conventions, but they must extend these conventions.
-
-[JSONSchema]:https://json-schema.org/
-[bf]:https://github.com/microsoft/botframework-cli
-[myget]:https://botbuilder.myget.org/gallery
-[runbot]:../runbot/readme.md
-[composer]:https://github.com/Microsoft/BotFramework-Composer
-[vscode]:https://code.visualstudio.com/Download
\ No newline at end of file
+# Relevant docs
+
+- [Full documentation](https://github.com/microsoft/BotBuilder-Samples/tree/master/experimental/generation/generator)
+- [Setup & get started](https://github.com/microsoft/BotBuilder-Samples/tree/master/experimental/generation/generator/docs/get-started.md)
+
+# Commands
+
+<!-- commands -->
+
+- [bf dialog:generate](#bf-dialoggenerate)
+<!-- - [bf dialog:integrate](#bf-dialogintegrate) -->
+
+## `bf dialog:generate`
+
+```
+USAGE
+
+`bf dialog:generate`
+
+OPTIONS
+
+- **--force, -f** Force overwriting generated files. (optional)
+- **--help, -h** Generate help. (optional)
+- **--locale, -l** Locales to generate. (optional, default: en-us)
+- **--output, -o** Output directory. (optional)
+- **--schema, -s** Path to your app.schema file. (required)
+- **--templates, -t** Directories with templates to use for generating assets. First definition wins.  A directory of "standard" includes the standard templates included with the tool.  You can also use the "template:<file>" URI to refer to files found in template directories.
+- **--verbose, -v** Verbose logging of generated files. (optional, default: standard)
+
+EXAMPLES
+
+  bf dialog:generate -s {SCHEMA FILE} -o {OUTPUT LOCATION}
+
+```
+
+_See [source code](src/commands/generate.ts).
+
+<!-->
+## `bf dialog:integrate`
+
+asdf
```

Comment:
???

Reply:
This entire section was supposed to be commented out - I wasn't sure if you wanted to document the command as-is, or wait for the integrated --integrate switch?

Reply:
removed